fast-list.js

```diff
@@ -404,7 +412,7 @@ FastList.prototype = {
   },
 
   reloadData: function() {
-    return this.rendered = schedule.mutation(function() {
+    return schedule.mutation(function() {
```

Comment:
Decided redefining `rendered` was kinda confusing as promises are meant to be a one time event. This function returns a Promise so user is still able to observe that if they wish to know when `reloadData` has completed.